dracut.sh

```diff
@@ -1128,6 +1128,10 @@ readonly DRACUT_TMPDIR="$(mktemp -p "$TMPDIR/" -d -t dracut.XXXXXX)"
     exit 1
 }
 
+# Cache file used to optimize get_maj_min()
+readonly get_maj_min_cache_file="${DRACUT_TMPDIR}/majmin_cache"
```

Comment:
Hmm, this probably need an additional `export`

```suggestion
declare -x -r get_maj_min_cache_file="${DRACUT_TMPDIR}/majmin_cache"
```

Reply:
I don't see any difference without or without the export.